packages/components/src/components/contract-card/contract-card-items/turbos-card-body.tsx

```diff
@@ -8,6 +7,45 @@ import Icon from '../../icon';
 import MobileWrapper from '../../mobile-wrapper';
 import Money from '../../money';
 import { ResultStatusIcon } from '../result-overlay/result-overlay.jsx';
+import { TContractInfo, TGetContractUpdateConfig } from '@deriv/shared/src/utils/contract/contract-types';
+
+type TToastConfig = {
```

Comment:
It's better not to keep all these types inside the file with the component, but it's a temporary solution until a proper folder won't bee completely migrated to TS (TSX)